app/cells/members/table_cell.rb

```diff
@@ -1,11 +1,11 @@
 module Members
   class TableCell < ::TableCell
-    options :authorize_update, :available_roles
-    columns :lastname, :firstname, :mail, :roles, :groups, :status
-    sortable_columns :lastname, :firstname, :mail
+    options :authorize_update, :available_roles, :is_filtered
+    columns :name, :mail, :roles, :groups, :status
+    sortable_columns :name, :mail
```

Comment:
Probably not an issue as I have never heard it to be one before but it is a little bit strange that one cannot sort by status.